src/App.css

```diff
@@ -61,6 +61,44 @@
    vertical-align: middle;
 }
 
+.delete-button {
+  margin-left: 10px;
+  width:  48px;
+  height: 48px; 
+  display: flex;
+  border-radius: 10px;
+}
+
+.new-task-container{
+  margin-top: 50px;
+  display: flex;
+  width: 770px;
+  height: 67px;
+  text-align: left;
+}
+
+
+.newtask-name {
```

Comment:
since you name the container `new-task-container`, I suggest renaming this and `newtask-button` to `new-task-name` and `new-task-button` for consistency